### PR TITLE
Footer and sidenav edits

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

File: _components/sidenav.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 layout: styleguide
 type: component
 title: Side navigation
-lead: "Hierarchical, vertical navigation to place at the side of a page. Note: we are currently developing horizontal navigation and headers for the top of a page"
+lead: Hierarchical, vertical navigation to place at the side of a page. Note: We're currently developing horizontal navigation and headers for the top of a page.
```
Comment (maya) [inline _components/sidenav.md]: @awfrancisco we need quotes here, bc jekyll requires this in frontmatter when there's a `:`. cc @carodew

Comment (awfrancisco) [inline _components/sidenav.md]: None of the other section have quotes on the lead.

Comment (maya) [inline _components/sidenav.md]: @awfrancisco see my comment.

Comment (awfrancisco) [inline _components/sidenav.md]: Got it. I was looking at the wrong colon. Will fix.

**Pull request merged**